docker-compose.yml

```diff
@@ -7,6 +7,11 @@ services:
       - '5432'
     env_file:
       - postgres.env
+    ports:
+    - "25432:5432"
```

Comment:
We should probably have separate dev and prod configurations eventually since I don't think it will be necessary to expose this port on production

Reply:
It is still listening to 127.0.0.1 and behind the firewall
I've been trying to add production config, but there is a problem
My original plan was to run 
`docker-compose -f docker-compose.yml -f docker-compose-production.yml up --build -d`

And nginx in first config is listening to port `80`  it will try to bind there at first and only then to port in next config